renderer/components/ui/ui.test.tsx

```diff
@@ -0,0 +1,124 @@
+import TimeBadge, { TimeBadgeProps } from "./TimeBadge";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+
+const day = 60 * 60 * 24 * 1000;
+const today = new Date();
+const yesterday = new Date(today.getTime() - day);
+
+const yesterdayLess6MockedProps: TimeBadgeProps = {
+  hours: 5,
+  startTime: "08:00",
+  selectedDate: yesterday,
+};
+const yesterdayLess8MockedProps: TimeBadgeProps = {
+  hours: 7,
+  startTime: "08:00",
+  selectedDate: yesterday,
+};
+
+const todayMorningLess6MockedProps: TimeBadgeProps = {
+  hours: 5,
+  startTime: "10:00",
+  selectedDate: today,
+};
+const todayMorningLess8MockedProps: TimeBadgeProps = {
+  hours: 7,
+  startTime: "10:00",
+  selectedDate: today,
+};
+
+const todayEveningLess6MockedProps: TimeBadgeProps = {
+  hours: 5,
+  startTime: "02:00",
+  selectedDate: today,
+};
+const todayEveningLess8MockedProps: TimeBadgeProps = {
+  hours: 7,
+  startTime: "02:00",
+  selectedDate: today,
+};
```

Comment:
Is it necessary to indicate the part of the day? At least 2 hours is night :) Maybe it makes sense to think about a simpler name.